app/Livewire/Project/Application/General.php

```
@@ -146,9 +146,9 @@ protected function rules(): array
             'gitRepository' => 'required',
             'gitBranch' => 'required',
             'gitCommitSha' => ['nullable', 'string', 'regex:/^[a-zA-Z0-9][a-zA-Z0-9._\-\/]*$/'],
-            'installCommand' => 'nullable',
-            'buildCommand' => 'nullable',
-            'startCommand' => 'nullable',
+            'installCommand' => ValidationPatterns::shellSafeCommandRules(),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ValidationPatterns' in method 'rules'. (undefined)

(StaticAccess)